test/integration/golden_response/get_statvar_group_node/root_mtv_jpn.json

```diff
@@ -5,13 +5,13 @@
       "id": "dc/g/Demographics",
       "specializedEntity": "Demographics",
       "displayName": "Demographics",
-      "numDescendentStatVars": 4102
+      "numDescendentStatVars": 4144
     },
     {
       "id": "dc/g/Household",
       "specializedEntity": "Household",
       "displayName": "Household",
-      "numDescendentStatVars": 1000
+      "numDescendentStatVars": 1223
```

Comment:
@pradh Please confirm the increased children

Reply:
Yes, that's from cl/394700399